workshop/content/japanese/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "パイプラインの作成"
+weight = 130
++++
+
+## 空のパイプラインの定義
+パイプラインの基本を定義する準備が整いました。
+
+
+`pipeline_stack.py` を開き、以下のように編集します。
+
+{{<highlight python "hl_lines=13 18-30">}}
+from constructs import Construct
+from aws_cdk import (
+    Stack,
+    aws_codecommit as codecommit,
+    pipelines as pipelines,
+)
+from cdk_workshop.pipeline_stage import WorkshopPipelineStage
```

Comment:
I still don't think this is supposed to be here just yet 🙂 `from pipeline_stage import WorkshopPipelineStage`